editor/plugins/org.fusesource.ide.projecttemplates/template-sources/medium/eap-wildfly/camel-test-spring-fuse7.1/pom.xml

```diff
@@ -223,6 +223,22 @@
 				<version.maven.war.plugin>3.2.2</version.maven.war.plugin>
 			</properties>
 		</profile>
+        <profile>
+			<id>java17-build</id>
+			<activation>
+				<jdk>[17,)</jdk>
+			</activation>
+            <properties>
+                <version.maven.war.plugin>3.3.2</version.maven.war.plugin>
```

Comment:
is it really necessary? Won't it come from the jdk12+ profile?

Reply:
the upgrade of version solves problems of EAP templates with JDK 17 (see https://issues.redhat.com/browse/FUSETOOLS-3593)

I was not sure if upgrade also for lower JDK versions will not cause any issues so I decided to enable higher version of `maven.war.plugin` only for JDK17+

Reply:
it can be of course changed to 3.3.2 version in JDK12+ profile to cover all cases.. if you see this approach better I will need to retest if everything properly works on different JDKs.. I tested only with JDK11 and JDK17

Reply:
no, it is fine like that. Thanks for the explanation